lib/cli/github_settings/do_set_branch_protection.js

```diff
@@ -230,15 +311,22 @@ do_set_branch_protection.help = [
     '{{usage}}',
     '',
     '{{options}}',
-    'This sets branch protection for the repo as follows. WARNING that this',
```

Comment:
Here are some suggested tweaks I made while grokking this as a I reviewed:
https://gist.github.com/trentm/2e278c83a4b63a5706d86d6ee4b860cf

Changes in my diff:

- Drop `-l, --label` option from `set-branch-protection`. It was accidentally
  copied from `jr gh check` and isn't used.
- Set `helpCol: 20` for nicer option help layout.
- `--status` -> `--required-status-checks` to more closely match the language
  in the GitHub UI and API (https://developer.github.com/v3/repos/branches/#get-required-status-checks-of-protected-branch)
- I re-wrote the `--help` output as I was learning what this did. Hopefully that
  is okay. I feel it is clearer for *me* at least. :)
- version update in package-lock.json (I'm guessing you are using node 0.10
  for your dev?)

Take as you like.

Reply:
These are all great, thank you!